contracts/utils/draft-InteroperableAddress.sol

```diff
@@ -0,0 +1,253 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.26;
+
+import {Math} from "./math/Math.sol";
+import {SafeCast} from "./math/SafeCast.sol";
+import {Bytes} from "./Bytes.sol";
+import {Calldata} from "./Calldata.sol";
+
+/**
+ * @dev Helper library to format and parse https://github.com/ethereum/ERCs/pull/1002[ERC-7930] interoperable
```

Comment:
If we know we'll get a link, we may be a good idea to just add it.

```suggestion
 * @dev Helper library to format and parse http://eips.ethereum.org/EIPS/eip-7930[ERC-7930] interoperable
```

Reply:
```suggestion
 * @dev Helper library to format and parse https://ethereum-magicians.org/t/erc-7930-interoperable-addresses/23365[ERC-7930] interoperable
```